src/components/Table/Table.jsx

```diff
@@ -53,8 +55,31 @@ const toUnderline = (data, row, field, underline) => {
   return false;
 };
 
+const getParamFromURL = (key) => {
+  const params = querystring.decode(window.location.search.substring(1));
```

Comment:
isn't the function `parse`?  Is it aliased?

https://nodejs.org/api/querystring.html